backend/data/src/main/kotlin/io/tolgee/service/organization/OrganizationRoleService.kt

```diff
@@ -260,18 +261,45 @@ class OrganizationRoleService(
   }
 
   fun leave(organizationId: Long) {
-    this.removeUser(authenticationFacade.authenticatedUser.id, organizationId)
+    self.removeUser(authenticationFacade.authenticatedUser.id, organizationId)
   }
 
   @Transactional
   fun removeUser(
     userId: Long,
     organizationId: Long,
   ) {
-    val managedBy = getManagedBy(userId)
-    if (managedBy != null && managedBy.id == organizationId) {
+    if (!canRemoveUser(userId, organizationId)) {
       throw ValidationException(Message.USER_IS_MANAGED_BY_ORGANIZATION)
     }
+
+    forceRemoveUser(userId, organizationId)
+  }
+
+  @Transactional
+  fun removeOrDeactivateUser(
+    userId: Long,
+    organizationId: Long,
```

Comment:
It seems like passing any random `organizationId` (even of a non-existent org) will lead to successful deactivation.

Is it ok? Just asking/nitpicking, maybe that's fine